x.py

```diff
@@ -306,6 +320,15 @@ def test_tcl(dir: str, rest: List[str]) -> None:
     parser_test_tcl.add_argument('rest', nargs=REMAINDER, help="the rest of arguments to forward to TCL scripts")
     parser_test_tcl.set_defaults(func=test_tcl)
 
+    parser_test_go = parser_test_subparsers.add_parser(
+        'go',
+        description="Test kvrocks via go test cases",
+        help="Test kvrocks via go test cases",
+    )
+    parser_test_go.add_argument('dir', metavar='BUILD_DIR', nargs='?', default='build', help="directory including kvrocks build files")
```

Comment:
`require=True`?

Reply:
We treat `build` as a default directory for building and testing, e.g.
```
./x.py build [build]
./x.py test tcl [build]
./x.py test go [build]
```
`[build]` can be omitted.

Reply:
Oops. I omit the `default='build'` part.